src/pool-bin/BinPoolManager.sol

```diff
@@ -102,9 +102,9 @@ contract BinPoolManager is IBinPoolManager, Fees, Extsload {
         Hooks.validateHookConfig(key);
         _validateHookNoOp(key);
 
-        /// @notice init value for dynamic swap fee is 0, but hook can still set it in afterInitialize
-        uint24 swapFee = key.fee.getInitialSwapFee();
-        if (swapFee.isSwapFeeTooLarge(SwapFeeLibrary.TEN_PERCENT_FEE)) revert FeeTooLarge();
+        /// @notice init value for dynamic lp fee is 0, but hook can still set it in afterInitialize
+        uint24 lpFee = key.fee.getInitialLPFee();
+        lpFee.validate(LPFeeLibrary.TEN_PERCENT_FEE);
```

Comment:
Confirm: are we still keeping 10% for bin pool lp fee ? 
It's in fact  at max `(1 - 0.1%) * 10% = 9.99%` charged on input token compare to previous

Reply:
aren't we setting max fee to 100% just like uni?

Reply:
"aren't we setting max fee to 100% just like uni?"

Before We were just following the original traderjoe behavior

Reply:
> Confirm: are we still keeping 10% for bin pool lp fee ?
let's create an issue to track this -- so we can answer there and pass the issue link to future devs when they ask about why joe is 10% limit. On a side note, so long this PR does not change the current max 10% fee behavior, then its alright. 

> It's in fact at max (1 - 0.1%) * 10% = 9.99% charged on input token compare to previous
can you explain a bit more? you mean before this PR the fee is 10% and somehow now is 9.99%? or previously and now is 9.99% ?

Reply:
"can you explain a bit more? you mean before this PR the fee is 10% and somehow now is 9.99%? or previously and now is 9.99% ?"

sorry not 9.99%, it should be 10.09%

I mean before the PR the fee is 10% because what we set was actually swap fee which includes protocol fee already. But after this PR we are setting lp fee here, although it can still be 10%, but since it's charged after protocol, on swap fee perspective it's at max (saying 0.1% protocol fee )  `0.1%  + (1 - 0.1%) * 10% = 10.09% `